wiki/Ranking_Criteria/en.md

```diff
@@ -153,22 +156,22 @@ This category contains explicit allowance statements of concepts and rules that
 - **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** If there are 3 or more contributing artists and they are not part of one officially labelled group, `Various Artists` or other descriptive artist labels may be used instead.
-- **For Remixes/Covers, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is remixed.** This marker should all be in parentheses and contain the Remix/Cover artist followed by descriptor.
+- **For remixes/covers, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is remixed.** This marker should all be in parentheses and contain the remix/cover artist followed by descriptor.
 
 ## Timing
 
 ### Rules
 
-- **[Uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) must be used to accurately beatmap the song's time signatures.** If an incorrect time signature lasts for more than one bar, a uninherited timing point must be added on the next downbeat to reset the time signature. For #/4-signatures unsupported by the editor, metronome resets or editing of the `.osu` file are acceptable. For other unsupported time signatures, refer to this [exemplary chart](/wiki/shared/timing/Timing_signature_reference_chart.png), and see [this guide](/wiki/Guides/Timing_Songs_With_8-Signatures) for further information.
-- **Beatmaps must be perfectly timed.** This means [BPM](/wiki/Beatmapping/Beats_per_minute) and [offset](/wiki/Beatmapping/Offset) are exactly synchronised with the song. Beatmaps with constantly changing BPM may be impossible to perfectly time and should instead be as accurate as possible without negatively affecting gameplay. Complex timing during breaks or spinners is optional.
+- **[Uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) must be used to accurately beatmap the song's time signatures.** If an incorrect time signature lasts for more than one bar, an uninherited timing point must be added on the next downbeat to reset the time signature. For #/4-signatures unsupported by the editor, metronome resets or editing of the `.osu` file are acceptable. For other unsupported time signatures, refer to this [exemplary chart](/wiki/shared/timing/Timing_signature_reference_chart.png), and see the [Timing songs with #/8-signatures](/wiki/Guides/Timing_Songs_With_8-Signatures) guide for further information.
+- **A beatmap must be perfectly timed.** This means [BPM](/wiki/Beatmapping/Beats_per_minute) and [offset](/wiki/Beatmapping/Offset) are exactly synchronised with the song. Beatmaps with constantly changing BPM may be impossible to perfectly time and should instead be as accurate as possible without negatively affecting gameplay. Complex timing during breaks or spinners is optional.
```

Comment:
disagree

Reply:
could you please be more specific? if anything it's my attempt to avoid RC switching between "Beatmaps must ..." and "A beatmap must ..." back and forth

Reply:
there are [more](https://github.com/ppy/osu-wiki/blob/0b292efbc04e39bb4e14a956b3a50ae724d06098/wiki/Ranking_Criteria/en.md?plain=1#L62) [instances](https://github.com/ppy/osu-wiki/blob/0b292efbc04e39bb4e14a956b3a50ae724d06098/wiki/Ranking_Criteria/en.md?plain=1#L29) of this if that's what you're going for

if anything, the style would need to be consistent between list items in a list, though it would make the asc very repetitive.

however, this doesn't only apply to beatmaps. singular and plural are used interchangeably for all list items. with that the case, i'd only worry about what makes sense and which one sounds better, and to me plural is better here (while being slightly more consistent with above and below items).

Reply:
alright